android/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name" translatable="false">FOSSASIA 2017</string>
+    <string name="app_name" translatable="false">Open Event</string>
```

Comment:
The generator searches for `OpenEvent` instead of `Open Event` while generating apk, so please either change it in the generator or here